absl/synchronization/internal/kernel_timeout.h

```diff
@@ -117,9 +114,11 @@ class KernelTimeout {
   // Windows. Callers should recognize that the return value is a
   // relative duration (it should be recomputed by calling this method
   // in the case of a spurious wakeup).
-  DWORD InMillisecondsFromNow() const {
+  typedef unsigned long dword;
```

Comment:
Please also update the comment.

Reply:
Added comments about why we define out own `DWORD` and `INFINITE` instead of including `<intsafe.h>`.